### ISSUE TITLE
Crosslink new macOS Mac Catalyst/Safari guidance

### ISSUE THREAD

**Issue by guardrex** [dotnet/AspNetCore.Docs#28109]
Per https://github.com/dotnet/docs-maui/pull/1197.

Thanks @davidbritch ... I think I can pull the following line ...

> Using browser developer tools with Mac Catalyst apps isn't currently supported. You may wish to run the \[iOS](https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?pivots=ios&view=aspnetcore-7.0) application instead, as the behavior is similar between iOS and Mac Catalyst.

... from https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?view=aspnetcore-7.0&pivots=macos

... and replace it with a crosslink to the new guidance that you placed ... something like ...

> To inspect a Blazor Hybrid app on Mac Catalyst, see \[Host a Blazor web app in a .NET MAUI app using BlazorWebView](/dotnet/maui/user-interface/controls/blazorwebview#inspect-a-blazor-hybrid-app-on-mac-catalyst).

Does that seem sufficient to you?

The alternative is that I copy and paste the section, but that makes for the possibility that something would shift in a future release that wouldn't get picked up on this end for a stale bit of guidance 😈.

If we go with that line ☝️, I'll mark the PR ***DRAFT*** and keep an eye on the live doc over there to make sure it doesn't merge until your new guidance goes live before merging the PR.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 48e7c079-a7e4-e0a4-ed19-935436113b70
* Version Independent ID: 48e7c079-a7e4-e0a4-ed19-935436113b70
* Content: [Use browser developer tools with ASP.NET Core Blazor Hybrid](https://learn.microsoft.com/en-us/aspnet/core/blazor/hybrid/developer-tools?view=aspnetcore-7.0)
* Content Source: [aspnetcore/blazor/hybrid/developer-tools.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/blazor/hybrid/developer-tools.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
# NOTE TO SELF

Move the line ***UP***, too. Place it before the NOTE.

**Comment by guardrex** [comment]
@davidbritch ... I actually had a whole piece of this topic commented-out. It reads like this ...

```
To use Safari developer tools with a macOS app:

1. Open desktop Safari.
1. Select the **Preferences** > **Advanced** > **Show Develop** menu command.
1. Run the .NET MAUI Blazor app in the macOS simulator.
1. Return to desktop Safari. Select **Develop** > **Simulator** > **0.0.0.0**. If multiple entries for **0.0.0.0** are present, select the entry that highlights the <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView>. The <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView> is highlighted in blue in the macOS simulator when the correct **0.0.0.0** entry is selected.
1. The **Web Inspector** window appears for the <xref:Microsoft.AspNetCore.Components.WebView.Maui.BlazorWebView>.
1. Developer tools provide a variety of features for working with apps, including which assets the page requested, how long assets took to load, and the content of loaded assets. The following example shows the **Console** tab, which includes any exception messages generated by the framework or developer code:

   ![Safari Web Inspector for a Blazor Hybrid app](~/blazor/hybrid/developer-tools/_static/safari2.png)
```

I'll research and see what's what with that if you don't know that it's right or wrong guidance.

**Comment by guardrex** [comment]
Yeah .... mmmmmmm 🤔 ... Seems like I should fire up the MacBook tomorrow and confirm the guidance with the piece that you provided. If it all works as planned, I ***would copy your section over here**. If you keep your section where you have it, it seems like we'll just need to have duplicate coverage on this one. This pivot is going to explain how to configure and run completely without the dev needing to navigate away to make it work properly.

**Comment by davidbritch** [comment]
@guardrex Happy to surrender it to your docs. They should be the source of truth for Blazor hybrid.

Your step list looks right, but I think it needs some terminology updates.

**Comment by guardrex** [comment]
... one more note ...

I retitled the Safari help link. I've moved all of the additional resource links to the bottom of the page.

The Safari help link goes to the doc that you cited, and it leads to everything else they have via its *Table of Contents* ...

[Safari Developer Help](https://support.apple.com/guide/safari-developer/welcome/mac)

My concern about that direct link to the overview is that with that code segment stuck in the URL it looks unstable ...

https://support.apple.com/guide/safari-developer/safari-developer-tools-overview-dev073038698/mac

Do you have any concerns about "dev073038698" weird path segment portion? If not, I'll change the link to that because the landing page loads actual content. It is a better landing page. It just scares me a little in terms of the link breaking at some point in the future.
